docker/Dockerfile.build-linux

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.4-slim-stretch
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends cmake make gcc pkg-config squashfs-tools git curl bison \
+    && rm -rf /var/lib/apt/lists/* \
+    && curl -L http://enclose.io/rubyc/rubyc-linux-x64.gz | gunzip > /usr/local/bin/rubyc \
```

Comment:
Just a thought, downloading a binary from a non-TLS URL could make some users uncomfortable.  How widely used is this Dockerfile expected to be used -- just for internal development or will ordinary licensed users run this too?

It looks like `enclose.io` sadly doesn't have a valid TLS cert.  Is there an HTTPS URL this could be downloaded from?

Not a big diff but I'd put the rubyc curl in a separate RUN line so it's cached separately from the apt-get (in fact maybe even before it so the curl will still be cached when the apt-get changes).

Reply:
> How widely used is this Dockerfile expected to be used -- just for internal development or will ordinary licensed users run this too?

Only for internal development / building packages.  General usage will download the built packages from a GitHub release which is HTTPS 👍 

> It looks like enclose.io sadly doesn't have a valid TLS cert. Is there an HTTPS URL this could be downloaded from?

😢 I don't think so.  I'll file an issue at the repo to ask them to set up TLS.

Reply:
> Not a big diff but I'd put the rubyc curl in a separate RUN line so it's cached separately from the apt-get (in fact maybe even before it so the curl will still be cached when the apt-get changes).

👍 to separating them.  I'd expect rubyc to be updated more frequently than any of the apt-get requirements, so it should probably come after.